helion/runtime/settings.py

```diff
@@ -72,6 +73,7 @@ class _Settings:
     allow_warp_specialize: bool = (
         os.environ.get("HELION_ALLOW_WARP_SPECIALIZE", "1") == "1"
     )
+    ref_mode: RefMode = RefMode.OFF
```

Comment:
Why do we have this, and the threadlocal, and the env var?  Wouldn't just one suffice?